api/taskcluster/webhook.go

```diff
@@ -160,9 +160,14 @@ func handleStatusEvent(ctx context.Context, payload []byte) (bool, error) {
 	// The default timeout is 5s, not enough for the receiver to download the reports.
 	slowCtx, cancel := context.WithTimeout(ctx, resultsReceiverTimeout)
 	defer cancel()
-	var labels []string
+	labels := mapset.NewSet()
 	if status.IsOnMaster() {
-		labels = []string{"master"}
+		labels.Add(shared.MasterLabel)
+	} else {
+		sender := status.GetSender().GetLogin()
```

Comment:
Where does this information come from in the API? https://api.github.com/repos/web-platform-tests/wpt/commits/7c6c6afac5e1f9ed9cae1159c3f8a58ad27ff9b8/statuses is for https://github.com/web-platform-tests/wpt/pull/14629#issuecomment-449174461 and it doesn't seem to include a sender. I guess it's only for PRs?

Reply:
Good catch actually; checks attach the sender (who triggered the suite, by pushing the commit).
Changed this to be `status.GetCommit().GetAuthor().GetLogin()` instead.